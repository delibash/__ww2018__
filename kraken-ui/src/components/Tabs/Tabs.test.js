import React from 'react';
import renderer from 'react-test-renderer'
import 'jest-styled-components'
import TabsComponent from './Tabs';

test('Tabs', () => {
  const tree = renderer.create(
    <TabsComponent
      tab={['Name of Tab']}
      tabPanel={[<section />]}
    />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});