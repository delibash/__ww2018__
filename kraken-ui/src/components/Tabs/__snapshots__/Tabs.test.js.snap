// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tabs 1`] = `
.c0 {
  margin: 0 -1.8rem;
}

.c0 .react-tabs__tab-panel {
  padding: 1.8rem;
  min-height: calc(100vh - 76px);
  margin-top: -.1rem;
  box-shadow: inset 0 0 0 .1rem rgba(0,0,0,.25);
}

.c0 .react-tabs__tab-list {
  border-bottom: .1rem solid rgba(0,0,0,.25);
  margin-bottom: 0;
}

.c0 .react-tabs__tab {
  border: 0;
  margin-left: 1.3rem;
  box-shadow: inset 0 0 0 .1rem rgba(0,0,0,.25);
}

.c0 .react-tabs__tab--selected {
  border-radius: .2rem .2rem 0 0;
  background-color: rgba(74,74,74,1);
  color: rgba(255,255,255,1);
}

<div
  className="c0"
>
  <div
    className="react-tabs"
    data-tabs={true}
    onClick={[Function]}
    onKeyDown={[Function]}
  >
    <ul
      className="react-tabs__tab-list"
      role="tablist"
    >
      <li
        aria-controls="react-tabs-1"
        aria-disabled="false"
        aria-selected="true"
        className="react-tabs__tab react-tabs__tab--selected"
        id="react-tabs-0"
        role="tab"
        tabIndex="0"
      >
        Name of Tab
      </li>
    </ul>
    <div
      aria-labelledby="react-tabs-0"
      className="react-tabs__tab-panel react-tabs__tab-panel--selected"
      id="react-tabs-1"
      role="tabpanel"
    >
      <section />
    </div>
  </div>
</div>
`;
